ansible_base/lib/serializers/common.py

```diff
@@ -36,6 +36,11 @@ def __init__(self, instance=None, data=empty, **kwargs):
     def get_url(self, obj) -> str:
         return get_url_for_object(obj)
 
+    @property
+    def is_list_view(self) -> bool:
+        view = self.context.get('view')
+        return view and view.action == 'list'
```

Comment:
If you don't get a view for some reason should an exception be raised or we are good with returning `False` in that condition?

Reply:
Ended up not needing this, so nixing this code.